### DISCUSSION TITLE
Mimicing the Hydro prompt?

### DISCUSSION THREAD

**Discussion by mattmc3** [starship/starship#3478] [Q&A]
Fish's [Hydro prompt](https://github.com/jorgebucaran/hydro) has a very interesting feature in that it changes the style of the final `$PWD` directory to be bold, and potentially even a different color. Something like "~/.config/**starship**". I've reviewed the [documentation for directory](https://starship.rs/config/#directory), but I cannot see a way to do this with Starship. Is there a way to further customize the directory style to achieve this?

**Comment by civiks** [comment]
Bump, also looking for the same
